cowprotocol/cowswap
# Update terms icons

# Summary

Added [new icons provided by Clement](https://drive.google.com/drive/u/1/folders/1fARDhJIKT4OdXxipw3ujutd-0L8U_BXo)

![Screenshot 2023-01-05 at 14 49 47](https://user-images.githubusercontent.com/43217/210838406-c845e44a-b9d8-43a0-914e-b1333901e7ae.png)

Also small fix based on [this comment](https://github.com/cowprotocol/cowswap/pull/1791#issuecomment-1365259375)

# To Test

1. Open the app, open the menu on top
* Terms and conditions, cookie policy and privacy policy should have their own unique icons
